umodbus/client/serial/rtu.py

```diff
@@ -223,3 +223,27 @@ def send_message(adu, serial_port):
         serial_port.read, expected_response_size - exception_adu_size)
 
     return parse_response_adu(response_error_adu + response_remainder, adu)
+
+
+async def async_send_message(adu, reader, writer):
```

Comment:
Would be nice to also have an example file using this function. It isn't completely clear to me how I would use this with an actual serial device, as I would with the non-asyncio `send_message` function.

Reply:
Sure, I just added one: `scirpts/examples/simple_asyncio_rtu_client.py`.